README.md

```diff
@@ -1,2 +1,12 @@
 # aspose-imaging-probe
 This repository is for relevant probation tasks
+
+## Keywords
+
+edit a photo, editing of pictures, edit an image, photo background editor, photo background change, resizing an image, resizing photos, photo sizing, picture resizer, image crop, crop photo, filter photo, online photo editor, online image editor, photo editing app, free photo editor
```

Comment:
@anms23 , also additional question, I see some keywords that does not have correspondence in the text, for example editor. Also, maybe information related to applications should be included? I do not see coverage for topic How to edit you image or photo online: ?

Reply:
I've fixed keyword usage:
https://github.com/aspose-imaging/aspose-imaging-probe/commit/87620b4e18177aa9a3f6a3c985a2e582a73a0f58

I have the article part about online text editor. I could extend this part:
https://aspose-imaging.github.io/aspose-imaging-probe/how_to_edit_photo_image_programmaticaly#how-to-edit-your-photo-or-image-online

Reply:
@anms23 , thanks, yes, I mean that part maybe not big to cover this topic, how do you think? Maybe it really need to be extended to answer task questions? Or such volume will be enought?

Reply:
@sam199008 , I've extended the topic "How to edit you image or photo online".

Reply:
@anms23 , Thanks. Can you also promote more of imaging apps (I mean links in task)?

Reply:
@sam199008 , this link is broken:
https://products.aspose.app/imaging/en/image-filter

Reply:
@sam199008 these two links looks the same:
https://products.aspose.app/imaging/photo-editor
https://products.aspose.app/imaging/image-editor